mungegithub/features/repo-updates.go

```diff
@@ -352,3 +358,16 @@ func (o *RepoInfo) Reviewers(path string) sets.String {
 	}
 	return peopleForPath(path, o.reviewers, false, o.EnableMdYaml)
 }
+
+// IsApproverlessFile takes a path to a file as input and returns a bool
+// indicating whether the specified file does NOT require approval
+func (o *RepoInfo) IsApproverlessFile(path string) bool {
+	dir, fn := filepath.Split(path)
+	dir = canonicalize(dir)
+	noApprovalNeeded, ok := o.approverless[dir]
+	if !ok {
```

Comment:
Test is useless here. `return noApprovalNeeded.Has(fn)` would return false anyway